ui/modal/modalPublishPreview/view.jsx

```diff
@@ -0,0 +1,178 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import { Form, FormField } from 'component/common/form';
+import { Modal } from 'modal/modal';
+import Card from 'component/common/card';
+import Tag from 'component/tag';
+import MarkdownPreview from 'component/common/markdown-preview';
+import { COPYRIGHT, OTHER } from 'constants/licenses';
+
+type Props = {
+  filePath: string | WebFile,
+  optimize: boolean,
+  title: ?string,
+  description: ?string,
+  channel: ?string,
+  bid: ?number,
+  uri: ?string,
+  contentIsFree: boolean,
+  fee: {
+    amount: string,
+    currency: string,
+  },
+  language: string,
+  licenseType: string,
+  otherLicenseDescription: ?string,
+  licenseUrl: ?string,
+  tags: Array<Tag>,
+  isVid: boolean,
+  ffmpegStatus: any,
+  previewResponse: PublishResponse,
+  publish: (?string, ?boolean) => void,
+  closeModal: () => void,
+  enablePublishPreview: boolean,
+  setEnablePublishPreview: boolean => void,
+};
+
+class ModalPublishPreview extends React.PureComponent<Props> {
+  onConfirmed() {
+    const { filePath, publish, closeModal } = this.props;
+    // Publish for real:
+    publish(this.resolveFilePathName(filePath), false);
+    closeModal();
+  }
+
+  resolveFilePathName(filePath: string | WebFile) {
+    if (typeof filePath === 'string') {
+      return filePath;
+    } else {
+      return filePath.name;
+    }
+  }
+
+  createRow(label: string, value: any) {
+    // Probably need to do the colon differently when in RTL mode. Don't want it to be part of the string
+    // to reduce redundant strings for translation.
+    return (
+      <tr>
+        <td>
+          {label}
+          {label ? ':' : ''}
```

Comment:
colons are an unnecessary here, and generally should be avoided outside of visually indistinct text